ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -28,82 +36,101 @@ type ReduxMappedProps = Readonly<{
 
 type Props = ReduxMappedProps & NavigationInjectedProps;
 
+type State = Readonly<{
+  isTransactionStarted: boolean; // it will be true when the user accepted to proceed with a transaction and he is going to display the datail of the transaction as receipt
+}>;
+
 const styles = StyleSheet.create({
-  rowStyle: {
-    paddingTop: 10
+  value: {
+    flex: 1,
+    flexDirection: "row"
   },
-  alignedRight: {
+  align: {
     textAlign: "right"
+  },
+  titleRow: {
+    justifyContent: "space-between"
   }
 });
 
 /**
- * > PREFIXES:
- *   - LABEL_COL_SIZE_*: prefix that represents
- *     the width of the "label" column (the description
- *     of the field)
- *
- *   - VALUE_COL_SIZE_*: prefix that represents
- *     the width of the "value" column (the actual
- *     contents of the field)
- *
- *
- * > SUFFIXES:
- *   - *_NARROW_LABEL: suffix that represents the cases
- *     where "label" column should be narrow
- *     (i.e. when the "value" column contains free text)
- *     Proportions: 1/3 : 2/3
- *
- *   - *_WIDE_LABEL: suffix that represents the cases
- *     where the "label" columnn should be wide (i.e. when
- *     the "value" column is narrow (it has either a number
- *     or a date/time, thus allowing additional space for
- *     the label)
- *     Proportions: 1/2 : 1/2
+ * Details of transaction
  */
-const LABEL_COL_SIZE_NARROW_LABEL = 1;
-const VALUE_COL_SIZE_NARROW_LABEL = 2;
+export class TransactionDetailsScreen extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      isTransactionStarted: true // TODO: implement the proper state control @https://www.pivotaltracker.com/n/projects/2048617/stories/158395136
+    };
+  }
+
+  /**
+   * It provide the currency EUR symbol
+   * TODO: verify if implement it furthermore (other currencies) or mantain the EUR notation @https://www.pivotaltracker.com/n/projects/2048617/stories/158330111
+   */
+  private getCurrencySymbol(currency: string) {
+    if (currency === "EUR") {
+      return "€";
+    } else {
+      return currency;
+    }
+  }
 
-const LABEL_COL_SIZE_WIDE_LABEL = 1;
-const VALUE_COL_SIZE_WIDE_LABEL = 1;
+  /**
+   * It sum the amount to pay and the fee requested to perform the transaction
+   * TO DO: If required, it should be implemented the proper algorithm to manage values from 10^13
```

Comment:
link to story - ALWAYS create stories for TODOs, if you don't they will be forgotten :)